Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -418,6 +418,23 @@ spec:
     }'
 ```
 
+## Holder Pod Deprecation
+
+Rook plans to deprecate CSI "holder" pods in Rook v1.16. Clusters with `csi-*plugin-holder-*` pods
+present in the Rook operator namespace must plan to set `CSI_DISABLE_HOLDER_PODS` to `"true"` after
+Rook v1.14 is installed and before v1.16 is installed by following the migration sections below.
+
+CephClusters that do not use `network.provider: multus` can skip to the
+[Disabling Holder Pods](#disabling-holder-pods) section.
+
+CephClusters that use `network.provider: multus` will need to plan the migration more carefully.
+Read the [Disabling Holder Pods with Multus and CSI Host Networking](#disabling-holder-pods-with-multus-and-csi-host-networking)
+section. Decide whether to use CSI host networking or not following the sections below.
```

Comment:
> Decide whether to use CSI host networking or not following the sections below.

Without holder pods, won't we always require CSI host networking? Or maybe I'm misinterpreting this sentence.

Reply:
`CSI_ENABLE_HOST_NETWORKING` is not required to be true. But we did talk about making another update after this PR to make that situation unsupported.

Reply:
Sounds good to follow up in a separate PR